Message.php

```diff
@@ -28,6 +28,7 @@ class Message extends BaseMessage
      * @var \Swift_Message Swift message instance.
      */
     private $_swiftMessage;
+    public $messageClass = "\Swift_Message";
```

Comment:
`messageClass` property is already reserved and pointing to `yii\swiftmailer\Message`.